cpp/src/io/json/json_column.cu

```diff
@@ -530,6 +554,7 @@ void make_device_json_column(device_span<SymbolT const> input,
   std::map<std::pair<NodeIndexT, std::string>, NodeIndexT> mapped_columns;
   // find column_ids which are values, but should be ignored in validity
   std::vector<uint8_t> ignore_vals(num_columns, 0);
+  std::vector<uint8_t> is_mixed_string_column(num_columns, 0);
```

Comment:
Do we mean "mixed type column" rather than "mixed string column"? Similarly throughout the code below.
```suggestion
  std::vector<uint8_t> is_mixed_type_column(num_columns, 0);
```